alpakaConfig.cmake

```diff
@@ -595,7 +595,10 @@ IF(ALPAKA_ACC_GPU_CUDA_ENABLE)
                     IF(((CMAKE_COMPILER_IS_GNUCXX AND CMAKE_CXX_COMPILER_VERSION VERSION_LESS 5.0) OR
                         (CMAKE_CXX_COMPILER_ID MATCHES "Clang" AND CMAKE_CXX_COMPILER_VERSION VERSION_LESS 3.8)) AND
                         CUDA_VERSION VERSION_LESS 9.0)
-                        MESSAGE(WARNING "${CMAKE_CXX_COMPILER_ID} ${CMAKE_CXX_COMPILER_VERSION} not support -G with CUDA <= 8! "
+                        MESSAGE(WARNING "${CMAKE_CXX_COMPILER_ID} ${CMAKE_CXX_COMPILER_VERSION} does not support -G with CUDA <= 8! "
+                                        "Device debug symbols NOT added.")
+                    ELSEIF(MSVC)
+                        MESSAGE(WARNING "${CMAKE_CXX_COMPILER_ID} ${CMAKE_CXX_COMPILER_VERSION} does not support -G with CUDA! "
```

Comment:
I am not yet 100% sure why this workaround is necessary. I did not have to do this locally. Maybe this is necessary because I never used a multi-architecture build locally (`ALPAKA_CUDA_ARCH="30;75"`)? I will try to investigate this later.

Reply:
It could be that the node is running out of memory if you activate `-G`. Another thing is if you activate `-G` sometimes you will get issues with shared memory. What kind of issue do you saw?

Reply:
Many many of those:
```
  AccNameTest.cpp
ptxas C : /Users/travis/AppData/Local/Temp/tmpxft_000007cc_00000000-10_AccNameTest.compute_30.ptx, line 1093; error : Type of argument does not match formal parameter '_' [C:\Users\travis\build\BenjaminW3\alpaka\build\test\unit\acc\acc.vcxproj]
  ptxas fatal   : Ptx assembly aborted due to errors
  AccNameTest.cpp
  CMake Error at acc_generated_AccNameTest.cpp.obj.Debug.cmake:279 (message):
    Error generating file
    C:/Users/travis/build/BenjaminW3/alpaka/build/test/unit/acc/CMakeFiles/acc.dir/src/Debug/acc_generated_AccNameTest.cpp.obj
```
Here is the log: https://travis-ci.org/BenjaminW3/alpaka/builds/513137513

I really think it could have something to do with the architecture because all the errors contain `_30.ptx`.

Reply:
for documentation VTK-m also saw the similar error message with gcc:  https://gitlab.kitware.com/vtk/vtk-m/issues/294